DirectEmployers/MyJobs
# PD-3817 updating full names on activities page

## What's this PR do?
A data migration to update full names on Activity page
## Screenshots (if appropriate)
![screen shot 2017-09-25 at 11 12 22 am](https://user-images.githubusercontent.com/26224950/30815884-7278ae30-a1e2-11e7-963d-e03e9592b74c.png)

## Where should the reviewer start?
Migration 0004 in myjobs
## How should this be manually tested?
In https://secure.my.jobs/manage-users/view/activities check that full names are being displayed for TJD, PRM and NUO
## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
